explorer-api-common/src/main/java/org/zowe/api/common/zosmf/services/AbstractZosmfRequestRunner.java

```diff
@@ -30,23 +31,45 @@
 import java.io.IOException;
 import java.net.URI;
 import java.net.URISyntaxException;
+import java.util.ArrayList;
+import java.util.List;
 import java.util.stream.IntStream;
 
 @Slf4j
 public abstract class AbstractZosmfRequestRunner<T> {
     
+    private ArrayList<Header> requestHeaders;
+    
+    public AbstractZosmfRequestRunner(List<Header> headers) {
+        this.requestHeaders = (ArrayList<Header>) headers;
+    }
+    
+    public AbstractZosmfRequestRunner() {
+        this.requestHeaders = new ArrayList<Header>();
+    }
```

Comment:
Some people would argue that having the field private and only being set in a single place is cleaner ie something like
```java
    private final ArrayList<Header> requestHeaders;
    
    public AbstractZosmfRequestRunner(List<Header> headers) {
        this.requestHeaders = (ArrayList<Header>) headers;
    }
    
    public AbstractZosmfRequestRunner() {
        this(Collections.emptyList());
    }
```
but that is optional, so happy to approve and up to you if you think it's nicer

Reply:
I agree this is cleaner, will implement thanks!